css/main.css

```diff
@@ -256,22 +415,63 @@ header .menu img {
   font-size: 1.7rem;
 }
 
+@media (min-width: 768px) {
+  .cards {
+    flex-direction: row;
+    gap: 2.4rem;
+    flex-wrap: wrap;
+    justify-content: space-between;
+  }
+
+  .cards__card {
+    width: 31%;
+  }
+
+  .backcard01:hover {
+    background-image: url(../images/imgplaceholderback01.jpg);
+    border: 2px solid #000;
+  }
+
+  .backcard02:hover {
+    background-image: url(../images/imgplaceholderback02.jpg);
+    border: 2px solid #000;
+  }
+
+  .backcard03:hover {
+    background-image: url(../images/imgplaceholderback03.jpg);
+    border: 2px solid #000;
+  }
+
+  .card__h3 {
+    margin-top: 17.2rem;
+  }
+
+  .card__btn {
+    display: none;
+  }
+
+  .card__btn:hover {
+    display: block;
+  }
```

Comment:
- [OPTIONAL] I appreciate that you implemented the changes requested by previous reviewers! and I noticed you're stuck with this little problem, so I'll put it as optional. To solve this issue you can hover on the card__container to show the card_btn and disappear the card__tags  💯
> For example 💡
```
  .card__container:hover + .card__tags {
    display: none;
  }

  .card__container:hover + .card__btn {
    display: block;
  }
``` 
> And remember to remove this 
```
  .card__btn:hover {
    display: block;
  }
```

For the hard work you've done, your project deserves to be approved, and I am sure that you will fix this minor issue.🙏